### PR TITLE
v3.3 Changelog

### PR DESCRIPTION
**Pull request by jasonblais**

### CODE REVIEW

**Comment by jasonblais** [comment]
Updated changelog with initial draft of highlights, improvements, bug fixes and known issues.

Will add compatability and database changes tomorrow, list of contributors by tomorrow or early Thursday.

Also awaiting an update from esethna.

File: source/administration/changelog.md [hunk 1]
```diff
@@ -2,9 +2,198 @@
...
+- Improved response time of status indicator changing between online/offline/away.
+- Added status indicators to the Direct Message and Channel member lists.
+- Added `@here` to mention users who are online.
+
+#### OAuth 2.0 Service Provider
+- Added support for OAuth 2.0 service provider, allowing external applications to authenticate API requests to Mattermost.
+- Added an option to enable automatic authorization of trusted OAuth 2.0 applications by the Mattermost server.
+
+#### Email Batching
+- Added an option to oombine mentions and direct messages received over a specified time period into a single email notification.
```
Comment (lindy65) [inline source/administration/changelog.md]: `oombine` > `combine`

File: source/administration/changelog.md [hunk 2]
```diff
@@ -2,9 +2,198 @@
...
+#### iOS app
+- Fixed issue where “Refresh/Log out” was appearing frequently.
+- Fixed issue where center channel appears blank after initial page load.
+- Keyboard is now closed when a user executes a search.
+
+#### Mobile (iOS and Android apps)
+- Enter key now creates a new line instead of sending the message.
+- Added links to the mobile apps in the welcome email, tutorial, and main menu.
+- Added a mobile landing page that informs users of the mobile app when they access the site on a mobile web browser.
+- Permalinks are now availabe on mobile.
```
Comment (lindy65) [inline source/administration/changelog.md]: `availabe' > `available`

File: source/administration/changelog.md [hunk 3]
```diff
@@ -2,9 +2,198 @@
...
+- Made it easier to click on the ... menu when in the right hand sidebar view.
+
+#### User Interface
+- Channel header is now added to the View Info modal.
+- Configured channel introduction to respect the full width and centred channel views.
+- Removed signup link from signin page if all signup methods are disabled.
```
Comment (lindy65) [inline source/administration/changelog.md]: `signin` > `sign in`

File: source/administration/changelog.md [hunk 4]
```diff
@@ -2,9 +2,198 @@
...
+- Added the ability to set different themes for each team.
+- Added a checkbox to apply theme settings to all teams of which you are a member.
+- Users disabled or removed from the LDAP server are now made “Inactive” in Mattermost (previously their sessions were revoked and could no longer log in, but their account status was not set to “Inactive”).
+- Added the ability to force migrating authentication methods.
+- LDAP `Bindusername` and `Bindpassword` fields in the System Console are now optional to support anonymous binding.
+
+### Bug Fixes
+
+- The behavior of setting for Link Previews in Account Settings is no longer reversed.
+- Hitting the URL of a private team you used to belong to now redirects properly
```
Comment (lindy65) [inline source/administration/changelog.md]: This sentence is missing a period at the end.

File: source/administration/changelog.md [hunk 5]
```diff
@@ -2,9 +2,198 @@
...
+- Search terms contained in hashtags are now highlighted in the search results.
+- Fixed an issue with quick typesetting on IE-11 and Edge.
+- Fixed an issue with uploading SAML certificates if the files were removed from `config.json`.
+- Multiple files can now be selected on the file upload dialog of the desktop app.
+- Fixed a scrolling issue with the channel switcher.
+- Fixed system messages showing a small empty white box.
+- Fixed a markdown formatting issue with multiple lists in a row.
+- Team Admins can no longer demote System Admins.
+- The channel header now respects the setting for Channel Display Mode.
+- The System Console no longers freezes if accessing via URL when not logged in.
```
Comment (lindy65) [inline source/administration/changelog.md]: `longers` > `longer`

File: source/administration/changelog.md [hunk 6]
```diff
@@ -2,9 +2,198 @@
...
+- Site Name is now restricted to 30 characters to avoid text overflow.
+- Error is no longer thrown when switching between teams in the System Console.
+- Invalid password error is thrown if System Admin resets a password to something that doesn't meet the specified password requirements.
+- Fixed the percentage loading indicator on the image preview modal.
+- File upload overlay now appears on Edge.
+- Maximum Users per team and Minimum Password Length now default to resonable values if a bad input is saved.
```
Comment (lindy65) [inline source/administration/changelog.md]: `resonable` > `reasonable`

File: source/administration/changelog.md [hunk 7]
```diff
@@ -2,9 +2,198 @@
...
+- Right-hand side now updates when a new profile picture is saved.
+- Channels in the Channel Switcher are sorted by their handle if their display name is identical.
+- Setting the length for mobile sessions is now fixed in the System Console.
+- The “Test Connection” button in the System Console > Notifications > Email section now properly uses the saved SMTP password.
+- System Admins no longer recieve a Javascript error if a new message is recieved while in the System Console.
```
Comment (lindy65) [inline source/administration/changelog.md]: `recieve` > `receive`

`Javascript` > `JavaScript`??

File: source/administration/changelog.md [hunk 8]
```diff
@@ -2,9 +2,198 @@
...
+-
+
+### Known Issues
+
+- Deleted messages don't disappear from the channel for the user who deleted the message, if the message was previously edited and right hand sidebar is open.
+- After receiving an error for creating a channel with one character, updated channel name is not saved.
+- A single collapsed preview re-opens after refresh.
+- Removed user from team still appears in DM list from the team.
+- Files sent in private chat to members in a different team are not accessible.
+- Youtube videos show as “Video not found” on Desktop App.
```
Comment (lindy65) [inline source/administration/changelog.md]: `Youtube` > `YouTube`

**Comment by jasonblais** [comment]
Thanks @lindy65!!

**Comment by esethna** [comment]
+1 tweaks added

**Comment by esethna** [comment]
Removed email batching and sign-up process PRs that got pushed to 3.4

**Comment by jasonblais** [comment]
Changed HA to beta, removed confirmation dialogs for integrations, added two UI improvements, added a known issues

**Pull request merged**